charter.md

```diff
@@ -2,47 +2,33 @@
 
 ## Introduction
 
-Digital credentials based on IETF standards have use cases ranging from personal credentials, such as drivers licenses and vaccination proofs, to business-to-business or business-to-government applications.
-One example is fraud and counterfeiting prevention in cross-border trade documents by protecting digital representations of mill test reports, bills of materials, bills of lading, or commercial invoices.
-
-These scenarios were addressed with registered claim names in JOSE and COSE, but the resulting solutions have lead to fragmented approaches, where each working group rediscovers the essential architecture and conventions necessary for issuers to make claims about subjects.
-
-In order to meet privacy, security, and sustainability objectives, digital credentials need to be designed with awareness of computation and storage constraints associated with their use cases.
-The SPICE WG focuses on a few explicit higher level objectives: clear semantics, proof schemes supporting both traceability and unlinkability, data minimization and selective disclosure.
-
-These objectives can be achieved by leveraging industry adopted standards and managing trade-offs between cutting-edge and well-established cryptography.
-
-As a guiding principle for the SPICE WG, claims that work well in JSON must work well in CBOR, without loss of semantics. Compact expressions of claims consume less resources and expose less attack surface, these properties are in support sustainable design.
-
-The SPICE WG will support digital credential formats leveraging existing IETF standards and IANA registries, and create a framework for consistently extending them to support stakeholders that are building compliance and automation systems based on industry adopted cryptography and protocols.
-
-Digital credentials are identity documents with attributes (public or private claims in IANA registries) about the identity. The identifiers referenced could be about any subject, but common examples include people, devices, assets, organizations and processes.
+Digital credentials are essential to identity, access control, and digitization use cases that are part of modernization efforts, such as digital licenses or certificates of origin.
+COSE and JOSE provide building blocks, such as identifiers and public and private claim structures, which enable interoperability and cryptographic agility, but which require profiling to render digitization easy, unambiguous and consistent.
```

Comment:
"private claim structures" could refer to the fact that they can encrypted, or could refer to them being non-standard.

Reply:
Since we refer to JOSE / COSE so much, I assumed it would refer to the registry topic.
A public claim is a registered one, private claims are not registered, and there are reserved blocks in COSE for them.

although I think it is helpful to leave room for addressing encryption use cases more thoroughly (as in not making encryption out of scope).

Reply:
```suggestion
IETF WGs and IRTF RGs have provided COSE and JOSE based building blocks.  Things like identifiers, claim structures, and confidentiality for personally identified information enable interoperability and cryptographic agility, however, they have required profiling in order to render digitization easy, unambiguous and consistent.
```

Reply:
lgtm (please expect some editorial changes though later on)